### PR TITLE
[Feat]: greet user with their name

### PR DESCRIPTION
**Pull request by anjali1102**
- Welcome user with their name (store in LocalStorage)
- Edit Name of user 

https://user-images.githubusercontent.com/56559378/179828236-8917e50d-a1fe-423b-92f7-ddc53005ac11.mp4

image

### CODE REVIEW

File: src/components/Weather/Weather.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import "./WeatherBody.css";
 const Weather = () => {
   const [lat, setLat] = useState(0);
```
Comment (mehulsatardekar) [inline src/components/Weather/Weather.js:6]: naming can be improved here use latitude and longitude

File: src/pages/MainPage/MainPage.css [hunk 2]
```diff
@@ -1,39 +1,105 @@
...
+  font-size: 100px;
+  text-shadow: 2px 2px rgb(177, 175, 175);
 }
 
-.quote_container {
-  font-size: 17px;
+.weather-wrapper {
+  position: absolute;
+  right: 20px;
+  top: 20px;
```
Comment (mehulsatardekar) [inline src/pages/MainPage/MainPage.css:31]: use rems units

**Review by mehulsatardekar — APPROVED** [review]
just one suggestion top left side of the globe text can be lower and the rest looks fantastic good work🚀🚀🚀

**Review by dev-enforced — APPROVED** [review]
The live preview is running well https://lily-git-greetwithname-anjali1102.vercel.app/
The name is getting stored in local storage.

**Review by bipin7yadav — APPROVED** [review]
GTG

**Pull request merged**